src/backends/torch/torchsolver.cc

```diff
@@ -251,6 +257,7 @@ namespace dd
         try
           {
             torch::load(*_optimizer, sstate, device);
+            swap_swa_sgd();
```

Comment:
swap_swa_sgd directly in Ranger::load?

Reply:
ok

Reply:
well, no in fact, see below : there is an explicit solver.train() that hides the swap